File: src/main/scala/spring/model/graph/Edge.scala
package spring.model.graph

import scala.beans.BeanProperty

/**
  * Created by Yan Doroshenko (yandoroshenko@protonmail.com) on 28.10.16.
  */
class Edge(@BeanProperty from: Node, @BeanProperty to: Node) {

}

object Edge {
  def apply(from: Node, to: Node) = new Edge(from, to)
}

File: src/main/java/cz/cvut/kbss/spipes/model/dto/Edge.java
package cz.cvut.kbss.spipes.model.dto;

/**
 * Created by Yan Doroshenko (yandoroshenko@protonmail.com) on 14.04.2017.
 */
public class Edge {
    private String id;
    private String source;
    private String target;

    public Edge(String id, String source, String target) {
        this.id = id;
        this.source = source;
        this.target = target;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getSource() {
        return source;
    }

    public void setSource(String source) {
        this.source = source;
    }

    public String getTarget() {
        return target;
    }

    public void setTarget(String target) {
        this.target = target;
    }
}
